src/test/regress/expected/multi_mx_master_update_node.out

```diff
@@ -0,0 +1,373 @@
+-- Test creation of mx tables and metadata syncing
+SELECT nextval('pg_catalog.pg_dist_placement_placementid_seq') AS last_placement_id
+\gset
+SELECT nextval('pg_catalog.pg_dist_groupid_seq') AS last_group_id \gset
+SELECT nextval('pg_catalog.pg_dist_node_nodeid_seq') AS last_node_id \gset
+SELECT nextval('pg_catalog.pg_dist_colocationid_seq') AS last_colocation_id \gset
+SELECT nextval('pg_catalog.pg_dist_shardid_seq') AS last_shard_id \gset
+SET citus.replication_model TO streaming;
+SET citus.shard_count TO 8;
+SET citus.shard_replication_factor TO 1;
+-- set sync intervals to less than 5s so wait_until_metadata_sync never times out
```

Comment:
This often breaks when running tests valgrind, maybe make the difference a bit more aggressive..

Reply:
Done. Increased the timeout to 15s and interval to 3s. Tested it for couple of times with valgrind.